apps/auth0-rules/src/__tests__/add-user-metadata.test.ts

```diff
@@ -93,6 +93,7 @@ const apiUser: UserMetadataResponse = {
   questions: [],
   role: 'Grantee',
   algoliaApiKey: 'test-api-key',
+  labs: [],
```

Comment:
i dont think we want to add labs to auth0 token - i already think we have too much information on it. You can easily omit this field in `UserMetadataResponse` type